cmd/machine-config-daemon/start.go

```diff
@@ -77,11 +81,15 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 		cb.MachineConfigClientOrDie(componentName),
 		cb.KubeClientOrDie(componentName),
 		daemon.NewFileSystemClient(),
+		ctx.KubeInformerFactory.Core().V1().Nodes(),
 	)
 	if err != nil {
 		glog.Fatalf("failed to initialize daemon: %v", err)
 	}
 
+	ctx.KubeInformerFactory.Start(ctx.Stop)
+	close(ctx.KubeInformersStarted)
```

Comment:
I think a comment here would be helpful. Seeing a closing of the channel without going through the informer docs seems odd.